### PR TITLE
Update SageMaker-ModelMonitoring.ipynb

### PR DESCRIPTION
**Pull request by romankoles**
A few changes to be in-sync with AWS SageMaker Python SDK 2.x

*Issue #, if available:*

*Description of changes:*

I had some issues with the model due to updates on AWS side, so I made a few changes to be in-sync with AWS SageMaker Python SDK 2.x

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: sagemaker_model_monitor/introduction/SageMaker-ModelMonitoring.ipynb [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "import boto3\n",
```
Comment (paul-cb) [inline sagemaker_model_monitor/introduction/SageMaker-ModelMonitoring.ipynb:43]: <p>Syntax error here. Two commas after <code>region_name</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

File: sagemaker_model_monitor/introduction/SageMaker-ModelMonitoring.ipynb [hunk 2]
```diff
@@ -126,13 +127,13 @@
    "source": [
     "from time import gmtime, strftime\n",
     "from sagemaker.model import Model\n",
-    "from sagemaker.amazon.amazon_estimator import get_image_uri\n",
+    "from sagemaker import image_uris\n",
     "\n",
     "model_name = \"DEMO-xgb-churn-pred-model-monitor-\" + strftime(\"%Y-%m-%d-%H-%M-%S\", gmtime())\n",
     "model_url = 'https://{}.s3-{}.amazonaws.com/{}/xgb-churn-prediction-model.tar.gz'.format(bucket, region, prefix)\n",
-    "image_uri = get_image_uri(boto3.Session().region_name, 'xgboost', '0.90-1')\n",
+    "image_uri = sagemaker.image_uris.retrieve('xgboost', boto3.Session().region_name,, '0.90-1')\n",
```
Comment (paul-cb) [inline sagemaker_model_monitor/introduction/SageMaker-ModelMonitoring.ipynb:134]: Syntax error here. Two commas after `region_name`

Comment (romankoles) [reply sagemaker_model_monitor/introduction/SageMaker-ModelMonitoring.ipynb:134]: good catch, thank you very much. I was developing and testing in a corp environment and just 'copy/paste' the code here. I will fix it within a minute

Comment (paul-cb) [reply sagemaker_model_monitor/introduction/SageMaker-ModelMonitoring.ipynb:134]: No problem! Let me know once you make the fix. Thanks Roman.

Comment (romankoles) [reply sagemaker_model_monitor/introduction/SageMaker-ModelMonitoring.ipynb]: Hi Paul, I just fixed the typo. Thank you again

**Review by paul-cb — CHANGES_REQUESTED** [review]

File: sagemaker_model_monitor/introduction/SageMaker-ModelMonitoring.ipynb [hunk 3]
```diff
@@ -186,10 +187,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from sagemaker.predictor import RealTimePredictor\n",
+    "from sagemaker.predictor import Predictor\n",
+    "from sagemaker.serializers import CSVSerializer\n",
     "import time\n",
     "\n",
-    "predictor = RealTimePredictor(endpoint=endpoint_name,content_type='text/csv')\n",
+    "predictor = Predictor(endpoint_name, serializer=CSVSerializer()\n",
```
[the remark this answers is not in the document]
Comment (romankoles) [reply sagemaker_model_monitor/introduction/SageMaker-ModelMonitoring.ipynb]: Just fixed it, thank you

**Review by paul-cb — APPROVED** [review]
Approved

**Review by paul-cb — CHANGES_REQUESTED** [review]
It seems to be breaking on line 229 now. 
```
---------------------------------------------------------------------------
Exception encountered at "In [7]":
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-7-9dc354b11722> in <module>
      2 current_endpoint_capture_prefix = '{}/{}'.format(data_capture_prefix, endpoint_name)
      3 result = s3_client.list_objects(Bucket=bucket, Prefix=current_endpoint_capture_prefix)
----> 4 capture_files = [capture_file.get("Key") for capture_file in result.get('Contents')]
      5 print("Found Capture Files:")
      6 print("\n ".join(capture_files))
TypeError: 'NoneType' object is not iterable
```

**Comment by romankoles** [comment]
it works properly on my side. Please check credentials or environment for the testing run

**Comment by paul-cb** [comment]
> it works properly on my side. Please check credentials or environment for the testing run

What kernel on Studio are you using to run this notebook?

**Comment by romankoles** [comment]
> What kernel on Studio are you using to run this notebook?

Hi Paul, I was using conda_python3 kernel in SageMaker. I am not using Studio yet

**Comment by paul-cb** [comment]
Thanks @romankoles for your response. Our notebooks need to be compatible on Studio for it to be merged. Can you kindly test it on Studio?

**Comment by romankoles** [comment]
Hi Paul, sorry for the delay, busy life. I will try to test it on Studio next week

**Comment by zhaoqizqwang** [comment]
Closing this PR since it was created a long time ago and didn't pass all the checks. Feel free to open a new pull request instead.

**Pull request closed without merging**